source-connector/src/test/kotlin/org/neo4j/connectors/kafka/source/Neo4jCdcSourceIT.kt

```diff
@@ -188,13 +189,23 @@ abstract class Neo4jCdcSourceIT {
   }
 }
 
-@KeyValueConverter(key = AVRO, value = AVRO) class Neo4jCdcSourceAvroIT : Neo4jCdcSourceIT()
+@KeyValueConverter(key = AVRO, value = AVRO, payloadMode = PayloadMode.EXTENDED)
+class Neo4jCdcSourceAvroExtendedIT : Neo4jCdcSourceIT()
 
-@KeyValueConverter(key = JSON_SCHEMA, value = JSON_SCHEMA)
-class Neo4jCdcSourceJsonSchemaIT : Neo4jCdcSourceIT()
+@KeyValueConverter(key = AVRO, value = AVRO, payloadMode = PayloadMode.COMPACT)
+class Neo4jCdcSourceAvroCompactIT : Neo4jCdcSourceIT()
 
-@KeyValueConverter(key = JSON_EMBEDDED, value = JSON_EMBEDDED)
-class Neo4jCdcSourceJsonEmbeddedIT : Neo4jCdcSourceIT()
+@KeyValueConverter(key = JSON_SCHEMA, value = JSON_SCHEMA, payloadMode = PayloadMode.EXTENDED)
+class Neo4jCdcSourceJsonSchemaExtendedIT : Neo4jCdcSourceIT()
 
-@KeyValueConverter(key = PROTOBUF, value = PROTOBUF)
-class Neo4jCdcSourceProtobufIT : Neo4jCdcSourceIT()
+@KeyValueConverter(key = JSON_SCHEMA, value = JSON_SCHEMA, payloadMode = PayloadMode.COMPACT)
+class Neo4jCdcSourceJsonSchemaCompactIT : Neo4jCdcSourceIT()
+
+@KeyValueConverter(key = JSON_EMBEDDED, value = JSON_EMBEDDED, payloadMode = PayloadMode.EXTENDED)
+class Neo4jCdcSourceJsonEmbeddedExtendedIT : Neo4jCdcSourceIT()
+
+@KeyValueConverter(key = JSON_EMBEDDED, value = JSON_EMBEDDED, payloadMode = PayloadMode.COMPACT)
+class Neo4jCdcSourceJsonEmbeddedCompactIT : Neo4jCdcSourceIT()
+
+@KeyValueConverter(key = PROTOBUF, value = PROTOBUF, payloadMode = PayloadMode.EXTENDED)
+class Neo4jCdcSourceProtobufExtendedIT : Neo4jCdcSourceIT()
```

Comment:
I think we need to document that `COMPACT` is not supported with Protobuf converters.

Reply:
yes, I already added that in the documentation.